### PR TITLE
fix(codewhisperer):  Fix up & down not moving cursor when suggestion active

### PR DESCRIPTION
**Pull request by leigaol**
## Problem
When pressing up or down when suggestion is active, the cursor won't go up or down.

## Solution
Let cursor go up or down after user pressing pressing up or down( meanwhile the suggestion is rejected).

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by justinmk3 — APPROVED** [review]

**Comment by leigaol** [comment]
A change log item has been added

File: .changes/next-release/Bug Fix-a8470482-60c4-4711-b967-e5b8c446a027.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fix a bug when pressing up or down does not move cursor when suggestion is active"
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-a8470482-60c4-4711-b967-e5b8c446a027.json:3]: - The "Bug Fix" type will already list the item as a fix. No need to repeat "fix a bug" in the description. Just describe the bug or feature.
- Without mentioning "CodeWhisperer", the description has very little context.

```suggestion
	"description": "CodeWhisper: pressing up or down does not move cursor when suggestion is active"
```

File: src/codewhisperer/activation.ts [hunk 2]
```diff
@@ -335,8 +335,14 @@ export async function activate(context: ExtContext): Promise<void> {
                 }
             }),
             Commands.register('aws.codeWhisperer.rejectCodeSuggestion', async e => {
-                if (vscode.window.activeTextEditor)
+                if (vscode.window.activeTextEditor) {
                     await InlineCompletion.instance.rejectRecommendation(vscode.window.activeTextEditor)
+                    if (e === 'up') {
+                        await vscode.commands.executeCommand('cursorUp')
+                    } else if (e === 'down') {
```
Comment (justinmk3) [inline src/codewhisperer/activation.ts:342]: if `e` is some other value that isn't handled here, at least should log `warn()`.

File: .changes/next-release/Bug Fix-a8470482-60c4-4711-b967-e5b8c446a027.json (hunk 1 above)
Comment (leigaol) [reply .changes/next-release/Bug Fix-a8470482-60c4-4711-b967-e5b8c446a027.json]: Thanks, the changelog item has been updated.

File: src/codewhisperer/activation.ts (hunk 2 above)
Comment (leigaol) [reply src/codewhisperer/activation.ts:342]: Thanks for your suggestion. I added log warn() when the commend argument is unexpected.

**Pull request merged**